sdk/servicebus/Microsoft.Azure.ServiceBus/src/Core/IReceiverClient.cs

```diff
@@ -63,6 +63,11 @@ public interface IReceiverClient : IClientEntity
         /// <remarks>Enable prefetch to speed up the receive rate.</remarks>
         void RegisterMessageHandler(Func<Message, CancellationToken, Task> handler, MessageHandlerOptions messageHandlerOptions);
 
+        /// <summary>
+        /// Unregister message handler from the receiver if there is an active message handler registered. 
+        /// </summary>
+        Task UnregisterMessageHandler();
```

Comment:
nit:  I believe this should have an `Async` suffix for the name to conform with similar members in this type.